flask/tests/test_groups.py

```diff
@@ -210,7 +210,7 @@ def test_create_group(test_database, client, login_as, minio_admin):
             "/api/groups",
             json={"group_code": "sk", "group_name": "Invalid s3 bucket name"},
         ).status_code
-        == 422
+        == 400
```

Comment:
Why did these two status codes change?

Reply:
It is specified in `schemas.py` that `group_name` must satisfy a regex that doesn't contain spaces:[ link](https://github.com/ccmbioinfo/stager/blob/659a93a6745f0a7f5dd734f7e9950e3b0900a742/flask/app/schemas.py#L92). 

Marshmallow input validation catches the error and throws 400 code here: [link.](https://github.com/ccmbioinfo/stager/blob/659a93a6745f0a7f5dd734f7e9950e3b0900a742/flask/app/groups.py#L183)